https://gitlab.freedesktop.org/fontconfig/fontconfig/-/merge_requests/223

From d4a8381169984246dbb14b7a810abd170d7d0afd Mon Sep 17 00:00:00 2001
From: Sam James <sam@gentoo.org>
Date: Sat, 2 Apr 2022 02:36:24 +0100
Subject: [PATCH] configure.ac: allow disabling docbook

Sometimes we might want to build docs (install man pages, etc)
but allow disabling use of docbook.

Bug: https://bugs.gentoo.org/310157
Signed-off-by: Sam James <sam@gentoo.org>
--- a/configure.ac
+++ b/configure.ac
@@ -736,7 +736,13 @@ dnl ===========================================================================
 # Let people not build/install docs if they don't have docbook
 #
 
-AC_CHECK_PROG(HASDOCBOOK, docbook2html, yes, no)
+AC_ARG_ENABLE(docbook,
+	[AS_HELP_STRING([--disable-docbook],
+		[Disable building docs with docbook2html (default: no)])],,)
+
+if test x$enable_docbook != xno; then
+	AC_CHECK_PROG(HASDOCBOOK, docbook2html, yes, no)
+fi
 
 AM_CONDITIONAL(USEDOCBOOK, test "x$HASDOCBOOK" = xyes)
 
